src/common/gtm.js

```diff
@@ -0,0 +1,31 @@
+import { getAppIdFallback } from './appId';
+import AppIdMap from './appIdResolver';
+import { getTokenList } from './utils/storageManager';
+
+const GTM = (() => {
+    const isGtmApplicable = () => Object.values(AppIdMap).includes(`${getAppIdFallback()}`);
+
+    const pushDataLayer = data => {
+        if (isGtmApplicable()) {
+            // eslint-disable-next-line no-undef
+            dataLayer.push({
+                ...data,
+            });
+        }
+    };
+
+    const setVisitorId = () => {
+        const tokenList = getTokenList();
+        if (tokenList.length > 0) {
+            pushDataLayer({ visitorId: tokenList[0].loginInfo.loginid });
```

Comment:
what if the client switched the account from the dropdown?

Reply:
> what if the client switched the account from the dropdown?

The token list in local storage reflects the same order as can be seen in the dropdown, so the `0` index will always be the account that's active. The `setVisitorId` function is called on both account switch & logout, so it would pass the correct `visitorId` to the data layer.